config/reloadable.go

```diff
@@ -101,10 +101,10 @@ func (c *config) DiscoverGateway() string {
 }
 
 // GroupID is the logical name for a group of org hosts
-func (c *config) GroupID() string {
+func (c *config) Group() string {
```

Comment:
what was the thought changing to `Group` half the places and `GroupID` elsewhere?

Reply:
nvm - not half places, just this function call

Reply:
The function and property cannot have the same name, I'll end up hanging groupName on a realtime object instead of the root config object to fix this (future update)

Reply:
actually I think I'll update that now, keep the api the same to the reloadable config instead of having to change that later